Breeze.UI/src/app/wallet/send/send.component.ts

```diff
@@ -83,9 +79,29 @@ export class SendComponent implements OnInit {
     this.sendForm.valueChanges
       .subscribe(data => this.onValueChanged(data));
 
+    this.feeSubscription();
+
     this.onValueChanged();
   }
 
+  feeSubscription() {
+    this.sendForm.controls['fee'].valueChanges.subscribe((value) => {
```

Comment:
This subscription is never unsibscribed, potential memory leak. Please unsubscribe in `onDestroy`